docker-compose.override.yml.template.multievent

```diff
@@ -45,6 +47,7 @@ services:
       - *plugin-config-file-1
       - *plugin-config-file-2
       - *plugin-config-file-3
+      - *plugin-list
```

Comment:
It might be easier at this point to just move the reference up one level and copy the whole environment section to every uber container.

Reply:
I don't expect most people to have to mess with the EVENTNAME compose project name, though, so it seemed like overkill to make everyone deal with it.

One thing I have considered is using an .env file instead of defining the config in the docker compose. That might be worth doing now and it might be even better in the future.